### ISSUE TITLE
Switch features that use jaxb to depend on the new jaxb 2.2 internal feature.

### ISSUE THREAD

**Issue by nmittles** [OpenLiberty/open-liberty#3323]
Once the jaxb 2.2 internal feature is ready, all features need to be scanned to see which use jaxb. These features will then need to be updated to pull in the private jaxb 2.2 feature, and no longer depend on the version of jaxb shipped with the JDK.  This is necessary for supporting Java 9 that does not include jaxb and to resolve some java2security issues.

Current known list of features that need to be updated:
jca*  --- see com.ibm.ws.jca.utils.xml.ra.RaResourceAdapter
beanvalidation*  --- needed for loading validation.xml's

**Comment by scottkurz** [comment]
Sounds like the batch-1.0 would be a good fit too...as it uses JAXB internally.

**Comment by aguibert** [comment]
closing this as a duplicate of #4351
